packages/fluent/scss/core/_variables.scss

```diff
@@ -12,6 +12,50 @@ $use-input-spinner-width: true !default;
 $use-input-spinner-icon-offset: false !default;
 $use-picker-select-width: true !default;
 
+// Spacing
+$kendo-spacer: 1em !default;
```

Comment:
I'm not really sure if we need the spacer. I guess we can add it once we decide that we needed in it for any of the components.